Packs/FiltersAndTransformers/Scripts/IfElif/IfElif.py

```diff
@@ -0,0 +1,76 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import json
+import re
+
+ARGS = demisto.args()
+CONTEXT = demisto.context()
+OVERRIDE_BUILTINS = {'__builtins__': None}
+CONTEXT_REGEX = re.compile('${([\S]+?)}')
```

Comment:
Do we not limit context to ascii? we can change `\S` to `\w` for better sanitization.

Reply:
outdated